yo/static_content.py

```diff
@@ -0,0 +1,61 @@
+from aiohttp import web
```

Comment:
VAPID keys need to be changed in webpush - that said, it can be pulled via an AJAX call.